app/controllers/feeds_controller.rb

```diff
@@ -22,19 +23,38 @@ def new
     @feed.clear_attribute_changes(%i[file_name podcast_id private slug])
   end
 
+  def get_apple_show_options(feed)
```

Comment:
If there is no apple show id set, and the API key is set, then this will call the API to get the list automatically so the user can pick.

Reply:
good point!

Reply:
done!